### DISCUSSION TITLE
CRYPTO PROMOTION BOOK (CRYPROBOOK)

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2045] [Ideas] [answered]
1) WHAT IS CRYPROBOOK?

One of the biggest problems in the cryptocurrency industry is the lack of a platform where users or customers can get unbiased and complete information about the cryptocurrencies they follow and want to research. To provide general and unbiased information for all cryptocurrencies within our body as "CRYPROBOOK". For this, our goal is to create a booklet for every cryptocurrency. Likewise, creating a more detailed booklet like "Whitepaper" but containing "Whitepaper". we named it "CRYPROBOOK" for it.

       2)Actually, we wanted to define a new concept as CRYPTOBOOK. And we want to create a platform where every project can promote itself on more solid foundations.
and we will ensure that each project creates a digital booklet, CRYPROBOOK.

2)What is the content of CRYPROBOOK and what information does it contain?

As CRYPROBOOK, we aim to be a leading intertwined project on both corporate and personal issues.

How Does?
It aims to be a fun and institutional company to be able to vote, chat, update weekly or monthly information from the responsible institution, draws of related Cryptocurrencies and these are intertwined. and creating a decent social platform to talk about a neutral social cryptocurrency regardless of the stock market.

**Comment by yolandadadada** [comment]
I think you meant to say coingecko

**Comment by Akrepkapo** [comment, marked as answer]
No

**Comment by Nguyenhong14** [comment]
Very good

**Comment by zeland2323** [comment]
Thanks
